invenio_records_resources/services/files/processors/image.py

```diff
@@ -34,7 +36,7 @@ def can_process(self, file_record):
         """Images can be processed."""
         if HAS_IMAGEMAGICK:
             ext = self.file_extension(file_record).lower()
-            return ext in [".jpg", ".jpeg", ".png", ".tif", ".tiff"]
+            return ext in RECORDS_RESOURCES_IMAGE_FORMATS
```

Comment:
```suggestion
            return ext in current_app.config["RECORDS_RESOURCES_IMAGE_FORMATS"]
```

Otherwise, any changes to the config wouldn't be picked up.